Problems/4153/ans_4153_JHP.py

```diff
@@ -0,0 +1,29 @@
+# https://www.acmicpc.net/problem/4153
+# 문제 : 파타고라스의 정의
+
+import sys
+
+def solve():
+    
+    while True:
+        
+        input = sys.stdin.readline
+        x, y, z = map(int,input().split())
```

Comment:
```suggestion
        point = list(map(int,input().split())).sort()
```
아래 보니까 전부 리스트 활용이던데 처음부터 처리해버리면 깔끔할듯.

Reply:
넵, 앞으로 참조하겠습니다.
아래 구현하다보니 List가 편할거 같아서 만든건데 위에 input 신경을 못 썼네요.